tests/unit/test-util/test.adb.js

```diff
@@ -622,6 +632,35 @@ describe('utils/adb', () => {
        });
   });
 
+
+  describe('checkOrCleanArtifacts', () => {
+    it('cleans the previously artifacts directory when condition true' +
+      ' but not removed earlier', async () => {
+      const adb = getFakeADBKit({
+        adbClient: {
+          readdir: sinon.spy(() => Promise.resolve([file])),
+          shell: sinon.spy(() => Promise.resolve('')),
+        },
+        adbkitUtil: {
+          readAll: sinon.spy(() => Promise.resolve(Buffer.from('1\n'))),
+        },
+      });
+      const adbUtils = new ADBUtils({adb});
+      const temp = '/sdcard/';
```

Comment:
temp doesn't seem to be used anywhere else bug in the next line, doesn't seem necessary to give it a name.

Reply:
Can't directly concatenate the strings thus this was required, and can't ise concat function as it will add sdcard in end of artifactDir

Reply:
not sure what you mean with tou can't use concat function? there is no real need to use different ways to concatenate strings (the String concat method and the + operator).

you can just compose all of them together using the template string syntax I mentioned in another review comment:
```
const part1 = "part1";
const part2 = "part2";
const part3 = "part3";
const aComposedString = `${part1}${part2}${part3}`;
```